### PR TITLE
add account management guide

### PR DESCRIPTION
**Pull request by meldra**

### CODE REVIEW

File: content/_guides/account.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+if you are unable to log in to the account but remember its nickname.
+
+You can change your current password to a new one with a command like
+the following in your IRC client:
+
+```irc
+/msg NickServ SET PASSWORD newpasswordhere
+```
+
+Simply change `yournewpassword` to something that is hard to guess. Once
```
Comment (internet-catte) [inline content/_guides/account.md:26]:
```suggestion
Simply change `newpasswordhere` to something that is hard to guess. Once
```
consistent with codeblock above

File: content/_guides/account.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+expire or can often be accessed by other people if they guess the right URL.
+
+## Grouping and un-grouping nicknames
+
+It is a good idea to group an alternative nickname for your client to use if
+your current one is in use (usually by another copy of your client during
+connection time outs.)
+
+Most clients use the default stored nickname with a `_` or a number at the
+end, so take a look at your network settings and see what it has stored.
```
Comment (internet-catte) [inline content/_guides/account.md:55]:
```suggestion
end, so take a look at your client's settings and see what it has stored.
```
maybe more clear?

File: content/_guides/account.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+To add a new nickname to your account, while logged in, run the following
```
Comment (internet-catte) [inline content/_guides/account.md:57]:
```suggestion
To add a new nickname to your account, while logged in, change your nickname to it and run the following
```

File: content/_guides/account.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+change that setting first.
+
+## Changing your account name
+
+The account name is your primary nickname associated with your account. It can
+be swapped out for any other nickname you have grouped.
+
+```irc
+/msg NickServ SET ACCOUNTNAME yourgroupednick
+
```
Comment (internet-catte) [inline content/_guides/account.md:92]:
```suggestion
```
empty line in codeblock

File: content/_guides/account.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+## Grouping and un-grouping nicknames
+
+It is a good idea to group an alternative nickname for your client to use if
+your current one is in use (usually by another copy of your client during
+connection time outs.)
+
+Most clients use the default stored nickname with a `_` or a number at the
+end, so take a look at your client's settings and see what it has stored.
+
+To add a new nickname to your account, while logged in, change your nickname to it and run the following
```
Comment (meldra) [inline content/_guides/account.md:57]:
```suggestion
To add a new nickname to your account, while logged in, change your nickname
to it and run the following
```

**Review by sario528 — APPROVED** [review]

**Pull request merged**